rcttc-queries.sql

```diff
@@ -0,0 +1,120 @@
+use tiny_theaters;
+
+-- Find all performances in the last quarter of 2021 (Oct. 1, 2021 - Dec. 31 2021).
+select * from performance;
+
+select
+show_name
+from performance
+where show_date between '2021-10-01' and '2021-12-31';
+-- List customers without duplication.
+select
+first_name,
+last_name
+from customer;
```

Comment:
The result is correct in this case, but it is always safe to use the DISTINCT keyword.